### PR TITLE
 Add R version 3.3.2 to checks for download_no_libcurl.

### PR DESCRIPTION
**Pull request by jacob-ogre**
For me (on OSX, R 3.3.2), this fixes the install / download problem referenced in #22.

Thank you for `webshot`!

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -165,7 +165,8 @@ available_port <- function(port) {
 # 403 for HEAD requests. See
 # https://stat.ethz.ch/pipermail/r-devel/2016-June/072852.html
 download <- function(url, destfile, mode = "w") {
-  if (getRversion() == "3.3.0" || getRversion() == "3.3.1") {
+  if (getRversion() == "3.3.0" || getRversion() == "3.3.1" ||
```
Comment (wch) [inline R/utils.R]: Do you mind changing it to just `getRversion() >= "3.3.0"`? I have a feeling that the issue isn't going to be fixed in R any time soon.

**Comment by wch** [comment]
Thanks!

**Comment by dhuebner** [comment]
@wch When do you plan to release a new version? Currently webshot::install_phantomjs() fails in R 3.3.2 with 403

**Comment by wch** [comment]
@dhuebner it will probably be a few weeks before a new release. But until then you can install the development version using devtools.

**Comment by dhuebner** [comment]
> a few weeks before a new release
@wch You mean the webshot 0.3.3 release? When is it planed?

**Comment by wch** [comment]
It will be a few more weeks.

**Comment by dhuebner** [comment]
@wch What about an approximate date? :)  January? February?

**Comment by wch** [comment]
I'm hoping by mid-January... although if my schedule magically clears up before then, I'll try to get it out sooner.

**Comment by dhuebner** [comment]
Thanks, Winston.

**Comment by wch** [comment]
@dhuebner I just released a new version to CRAN.

**Comment by dhuebner** [comment]
@wch Very cool thanks!

**Pull request merged**